Tests/FluentAssertions.Net45.Specs/Net45.Specs.csproj

```diff
@@ -84,6 +84,9 @@
     <Otherwise />
   </Choose>
   <ItemGroup>
+    <Compile Include="..\FluentAssertions.Net40.Specs\XmlElementAssertionSpecs.cs">
```

Comment:
I just noticed that the XML stuff is only in the .NET 4.5 version. Is there  a reason for this? The XDocument stuff is in the Shared project.